app/views/admin/configs/show.html.erb

```diff
@@ -1191,61 +1191,61 @@
           <div id="uxBodyContainer" class="card-body collapse hide" aria-labelledby="uxBodyContainer">
             <fieldset class="grid gap-4">
               <div class="crayons-fieldx">
-                <%= admin_config_label :feed_style %>
-                <%= admin_config_description Constants::SiteConfig::DETAILS[:feed_style][:description] %>
+                <%= admin_config_label :feed_style, model: Settings::Experience %>
```

Comment:
I tested this out locally and ran into a `NameError` error when I navigated to `/admin/config`: `uninitialized constant Settings::Experience`. That said, I was able to successfully access the SiteConfig and edit/update fields as expected after changing the model name (here and in all other necessary locations as well!) to the following 😊 :
```suggestion
                <%= admin_config_label :feed_style, model: Settings::UserExperience %>
```